packages/components/src/Page/Page.tsx

```diff
@@ -63,14 +64,15 @@ export function Page({
   moreActionsMenu = [],
 }: PageProps) {
   const pageStyles = classnames(styles.page, styles[width]);
-  const [titleBarRef, { width: titleBarWidth }] = useResizeObserver<
-    HTMLDivElement
-  >();
+  const [
+    titleBarRef,
+    { width: titleBarWidth = defaultSizes.large },
+  ] = useResizeObserver<HTMLDivElement>();
 
   const titleBarClasses = classnames(styles.titleBar, {
-    [styles.small]: titleBarWidth && titleBarWidth > 265,
-    [styles.medium]: titleBarWidth && titleBarWidth > 500,
-    [styles.large]: titleBarWidth && titleBarWidth > 750,
+    [styles.small]: titleBarWidth > defaultSizes.smaller,
```

Comment:
I wonder if exporting `defaultSizes` as `Sizes` or something but be nicer here… I'm not sure why but this feels a bit odd to me… Perhaps I need to think more, also happy to chat.